src/services/unix-files.service.ts

```diff
@@ -0,0 +1,57 @@
+import { StreamService } from '.';
```

Comment:
Avoid import from `index.ts`, it could be dangerous :smiling_imp: 
```suggestion
import { StreamService } from './stream.service';
```